src/test/ui/generic-associated-types/issue-88595.rs

```diff
@@ -8,7 +8,7 @@ trait A<'a> {
     // FIXME(generic_associated_types): Remove one of the below bounds
     // https://github.com/rust-lang/rust/pull/90678#discussion_r744976085
     where
-        'a: 'b, Self: 'a, Self: 'b;
```

Comment:
This bound was required because of a bug that has since been fixed. FIXME still stands for the two remaining though.